users/relaying.rst

```diff
@@ -38,6 +61,10 @@ smooth operation::
     Usage of relaysrv:
       -debug
             Enable debug output
+      -ext-address string
+            An optional address to advertising as being available on.
+            Allows listening on an unprivileged port with port forwarding from e.g.                                             
+            443, and be connected to on port 443.
```

Comment:
I guess this is an IP address plus port number. Perhaps:
An optional IP address and port number that is the real external address:port that the relay server can be contacted at. This might be a public IP address and port that is forwarded in to the relay server, or just a port that is being forwarded in the host itself. This also allows listening on an unprivileged port and forwarding port 443 to that unprivileged port, so user can still connect to port 443.